tasks/cnm-to-cma/cnm_to_cma.py

```diff
@@ -0,0 +1,189 @@
+"""lambda function used to translate CNM messages to CMA messages in aws lambda with cumulus"""
+
+import os
+import re
+from datetime import datetime, timezone
+from typing import List
+from cumulus_logger import CumulusLogger
+from cumulus_process import Process
+
+REGION = os.environ.get("REGION", "us-west-2")
+cumulus_logger = CumulusLogger("cnm2cma")
+
+
+class CNM2CMA(Process):
+    """
+    Message transformation class to transform CNM to CMA
+
+    Attributes
+    ----------
+    logger: logger
+        cumulus logger
+    config: dictionary
+        configuration from cumulus
+    """
+
+    def __init__(self, *args, **kwargs):
+        """class init function"""
+
+        # super().__init__(*args, **kwargs)
+        super(CNM2CMA, self).__init__(*args, **kwargs)
+        self.logger = cumulus_logger
+
+    def process(self):
+        """Main process to transform cnm to cma
+
+        Return
+        ----------
+        dict
+            Payload that is returned as the cma which is a dictionary with list of granules
+        """
+        # This is the payload
+        cnm = self.input
```

Comment:
There are no checks on the input from either the Process class or here. I feel like at a minimum we should check that our input adheres to the CNM message schema. Consider leveraging the Pydantic library [https://docs.pydantic.dev/latest/](https://docs.pydantic.dev/latest/) to help define inputs, outputs and models for validation.